packages/lesswrong/components/tagging/NewTagItem.tsx

```diff
@@ -33,14 +33,35 @@ const styles = (theme: ThemeType): JssStyles => ({
     ...tagBodyStyles(theme),
     marginBottom: 18,
   },
+  discussionButton: {
+    ...theme.typography.commentStyle,
+    ...theme.typography.body2,
+    ...theme.typography.uiStyle,
+    color: theme.palette.grey[700],
+    display: "flex",
+    '& svg': {
```

Comment:
This suggests something is going wrong. You should very rarely have to subset on children tag-types. 

Also, this whole class appears to be unused.

Reply:
Ah, yep. Got factored out into it's own component. Styling was copied from Ray's original, so uh, blame him for being a bad teacher. :P